src/pages/index.astro

```diff
@@ -1,55 +1,110 @@
 ---
-import EmailForm from "../components/Email-Form.astro";
 import Layout from "../layouts/Layout.astro";
-import Hero from "../components/Hero.astro";
-import Features from "../components/Features.astro";
-import Notification from "../components/Notification.astro";
-import Footer from "../components/Footer.astro";
-import "../styles/index.css";
 ---
 
 <Layout title="Purple Bubble">
-  <main>
-    <div>
-      <script
-        type="text/javascript"
-        src="https://unpkg.com/@lottiefiles/lottie-player@1.6.0/dist/lottie-player.js"
-      ></script>
+  <section><p>We are Purple Bubble</p></section>
+  <section>
+    <p>
+      Made up of teens from around the globe, we hope to create a new messaging
+      app that will change the way you think about privacy and anonymity. We
+      want to revolutionize the way messaging apps work and make it so that
+      instead of being the product, you're using a product that works for you.
+    </p>
 
-      <div class="home-container">
-        <Hero />
-        <div class="chevron-container">
-          <button class="chevron-button" id="chevron-button"></button>
-        </div>
-        <Features />
+    <p>
+      We're currently in the early stages of development and are looking for
+      people to help us test our app. If you're interested in helping us test
+      our app, please sign up for our mailing list below. (We promise not to
+      send you any spam, only important milestones which will result in less
+      than 4-5 emails a year)
+    </p>
+  </section>
+  <section>
+    <form
+      action="https://postal.hackclub.com/subscribe"
+      method="POST"
+      accept-charset="utf-8"
+    >
+      <label for="name">Name</label><br />
+      <input
+        type="text"
+        name="name"
+        id="name"
+        placeholder="R. M. Bubbler"
+        required
+      />
+      <br />
+      <label for="email">Email</label><br />
+      <input
+        type="email"
+        name="email"
+        id="email"
+        placeholder="bubbler@purplebubble.org"
+        required
+      /><br /><br />
+
+      <span>
+        <input type="checkbox" name="gdpr" id="gdpr" required />
+        <strong>Marketing permission</strong>: I give my consent to Purple
+        Bubble to be in touch with me via email using the information I have
+        provided in this form for the purpose of news, updates and marketing.
+        (Required by GDPR)</span
```

Comment:
is this needed? we aren't focused in europe

Reply:
Its kinda needed now because you never know who will find the stite

Reply:
we could probably merge it with the piece underneath though

Reply:
legally yes we do need this.

Reply:
is there anyway we could make it smaller? i like merging it.